### ISSUE TITLE
Possible bug of recurrent propagator example?

### ISSUE THREAD

**Issue by ET-Chan** [tkocisky/oxnn#3]
While connecting the units of the prediction made by the network and the targets

```
   local losses = r.S('loss')
   for i=len,2,-1 do
      r.E { { after_out[i-1], targets[i] },
            mod_crit,
            losses[1] }
   end
```

But, from my understanding the target of the output of first time step (i.e. after_out[len-1]) is the input of second timestep (i.e. targets[len-1])... Does that mean the code actually should be

```
   local losses = r.S('loss')
   for i  = len - 1 , 1 , -1 do
      r.E { { after_out[i], targets[i] },
            mod_crit,
            losses[1] }
   end
```

Correct me if I me wrong. Thanks.

**Comment by tkocisky** [comment]
Yes, you are right. You can submit a PR if you want, otherwise I'll fix it.

**Comment by ET-Chan** [comment]
I will do that.

**Comment by tkocisky** [comment]
Fixed with #4.
